setup.cfg

```diff
@@ -15,16 +15,22 @@ replace = {new_version}
 universal = 1
 
 [flake8]
-exclude = docs
-max-line-length = 88
-ignore = 
+exclude =
+	docs/
+	aicsimageio/vendor/
+ignore =
 	E203
 	W291
 	W503
+max-line-length = 88
 
 [aliases]
 test = pytest
 
 [tool:pytest]
 collect_ignore = ['setup.py']
-
+markers =
+    raises
```

Comment:
If you're using the latest version of `pytest-raises`, I don't think you should need to make this marker explicit. They addressed this issue recently 🙂

Reply:
Removed! Thanks for the info